besu/src/main/java/org/hyperledger/besu/cli/util/BesuCommandCustomFactory.java

```diff
@@ -38,12 +38,11 @@ public BesuCommandCustomFactory(final PluginVersionsProvider pluginVersionsProvi
       return (T) new VersionProvider(pluginVersionsProvider);
     }
 
-    final Constructor<T> constructor = cls.getDeclaredConstructor();
     try {
+      final Constructor<T> constructor = cls.getDeclaredConstructor();
```

Comment:
This logic is already been performed in `CommandLine.defaultFactory().create(cls)` ... whats the point of repeating it here?